libphobos/testsuite/lib/libphobos.exp

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2017 Free Software Foundation, Inc.
+# Copyright (C) 2017-2018 Free Software Foundation, Inc.
 
```

Comment:
You added a # in the empty line between comments in the other files.

Reply:
Yes, because the rest of the copyright notice has `#` instead of empty lines.